package.js

```diff
@@ -1,13 +1,13 @@
 Package.describe({
     name: 'risul:moment-timezone',
     summary: 'Timezone support for moment.js, packaged for Meteor',
-    version: '0.5.0_4',
+    version: '0.5.4',
```

Comment:
Always wrap upstream versions with local `_x` suffix. So one can change Meteor package for the same upstream version.